research/conditional/utils/model_utils.py

```diff
@@ -245,6 +248,46 @@ def get_expert_choice_args(args):
     }
 
 
+def get_expert_choice_with_parallel_ff_args(args):
+    assert args.granularity_expert_config, "Must use granularity config"
+    expert_choice_params = get_expert_choice_args(args)
+    n_experts = expert_choice_params["n_experts"]
+    expert_size = expert_choice_params["expert_size"]
+    top_k_fraction = expert_choice_params["topk_fraction"]
+
+    def calculate_effective_expert_dff(_expert_size, _n_experts, _topk_fraction):
```

Comment:
Here we will also be able to use function `get_expert_choice_args` from this PR https://github.com/llm-random/llm-random/pull/169

Reply:
I think I do not understand what exactly do you mean?

Reply:
Nevermind, I misunderstood something